Busybox fold does not accept -w greater than 10000:

  fold: number 80000 is not in 1..10000 range

--- a/tests/t6036-long-output-lines.sh
+++ b/tests/t6036-long-output-lines.sh
@@ -22,7 +22,7 @@
 	test_cmp testfile output
 "
 test_expect_success 'pdsh does not truncate even longer lines' "
-	dd if=/dev/urandom bs=1024 count=100 | $base64 | tr -d '\n' | fold -w80000 > testfile2 &&
+	dd if=/dev/urandom bs=1024 count=100 | $base64 | tr -d '\n' | fold -w10000 > testfile2 &&
         echo >>testfile2 &&
 	pdsh -w foo -N -Rexec cat testfile2 > output2 &&
 	test_cmp testfile2 output2
